api/DOMStringList.json

```diff
@@ -20,7 +20,7 @@
             "version_added": true
           },
           "ie": {
-            "version_added": true
+            "version_added": "10"
```

Comment:
This matches https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility, so seems right.